packages/amplify-schema-validator/src/validators/fields-passed-are-in-parent-model.ts

```diff
@@ -0,0 +1,64 @@
+import {
+    DocumentNode,
+    Kind,
+    ListTypeNode,
+    ListValueNode,
+    ObjectTypeDefinitionNode,
+    StringValueNode,
+  } from 'graphql';
+  import { InvalidDirectiveError } from '../exceptions/invalid-directive-error';
+  import { getObjectWithName } from '../helpers/get-object-with-name';
+  import { resolveFieldTypeName } from '../helpers/resolve-field-type-name';
+  
+  /**
+   * Validates that any of the fields passed in are in the parent model
+   *
+   * @param schema graphql schema
+   * @returns true
+   */
+  export const validateFieldInParentModel = (schema: DocumentNode): Error[] => {
+    const errors: Error[] = [];
+    const objectTypeDefinitions = schema.definitions.filter(
+      (defintion) => defintion.kind === Kind.OBJECT_TYPE_DEFINITION,
+    ) as ObjectTypeDefinitionNode[];
+    objectTypeDefinitions.forEach((objectTypeDefinition) => {
+      const directiveFields = objectTypeDefinition.fields?.filter((objectField) => objectField.directives?.find(
+        (directive) => directive.name.value,
+      ));
+
+      directiveFields?.forEach((directiveField) => {
+        const typeName = resolveFieldTypeName(directiveField.type);
+        if (!typeName) {
+          return;
```

Comment:
this line is not covered

Reply:
if there is not value in covering this case in unit tests, you can add `/* istanbul ignore next */` to ignore the uncovered line.

Reply:
Fixed